diff --git a/net/minecraft/block/BlockAir.java b/net/minecraft/block/BlockAir.java
index 12bb353..b619f9c 100644
--- a/net/minecraft/block/BlockAir.java
+++ b/net/minecraft/block/BlockAir.java
@@ -21,6 +21,7 @@ public class BlockAir extends Block
      */
     public EnumBlockRenderType getRenderType(IBlockState state)
     {
+        System.out.println("Test");
         return EnumBlockRenderType.INVISIBLE;
     }
 
